src/HomePage.sass

```diff
@@ -15,16 +15,6 @@
     flex-direction: row
     margin: 30px
 
-  .config-line
```

Comment:
Left area now looks weird, the button is not aligned and there is no margin between the divider and the statuses

Reply:
just removed the divider.
Also canceled the hover resize. I think for now it's not very helpful, even annoying.